glommio/src/executor/placement/mod.rs

```diff
@@ -184,6 +184,26 @@ impl PoolPlacement {
     pub fn generate_cpu_set(self) -> Result<CpuSetGenerator> {
         CpuSetGenerator::pool(self)
     }
+
+    /// Shrinks the pool placement policy to the first `len` placements.
+    ///
+    /// If `len` is greater than the number of placements in the pool, this has
+    /// no effect.
+    pub fn shrink_to(self, count: usize) -> Self {
```

Comment:
why would this be public?
This looks confusing to me as an API

Reply:
It's now private